app/controllers/articles_controller.rb

```diff
@@ -30,12 +31,14 @@ def feed
     set_surrogate_key_header "feed"
     set_cache_control_headers(10.minutes.to_i, stale_while_revalidate: 30, stale_if_error: 1.day.to_i)
 
-    render layout: false, locals: {
+    render layout: false, content_type: "application/xml", locals: {
```

Comment:
> your browser doesn't do any handling of the document

Since browsers aren't the main targets for rendering RSS feeds, I think this would be ok. But I have no issues with changing to `application/xml`, it is the most widely used content type for feeds after all.

For completness' sake, the feeds we generate are `application/rss+xml`:

```
❯ http --print=h https://dev.to/feed/
HTTP/1.1 200 OK
Accept-Ranges: bytes
Access-Control-Allow-Origin: *
Age: 370
Cache-Control: public, no-cache
Connection: keep-alive
Content-Encoding: gzip
Content-Length: 33656
**Content-Type: application/rss+xml; charset=utf-8**
Date: Tue, 15 Feb 2022 04:11:57 GMT
Etag: W/"c1b337e0d6746bc198d2d44ac319d669"
Referrer-Policy: strict-origin-when-cross-origin
Server: Cowboy
Strict-Transport-Security: max-age=31557600
Vary: Accept-Encoding, X-Loggedin
Via: 1.1 vegur, 1.1 varnish, 1.1 varnish
X-Accel-Expires: 600
X-Cache: HIT, MISS
X-Cache-Hits: 1, 0
X-Content-Type-Options: nosniff
X-Download-Options: noopen
X-Frame-Options: SAMEORIGIN
X-Permitted-Cross-Domain-Policies: none
X-Request-Id: 50c62775-0694-4dac-bc0c-c0b4ff1a2a30
X-Runtime: 0.194992
X-Served-By: cache-den8277-DEN, cache-qpg1283-QPG
X-Timer: S1644898317.934761,VS0,VE220
X-Xss-Protection: 1; mode=block
```